setup.py

```diff
@@ -12,12 +12,33 @@
 needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 
+# Check Python version
+if sys.version_info[:2] < (3, 6):
+    print('This package requires a verion of Python>=3.6.'
+          'Please upgrading your version of Python to use MembraneCurvature.')
+    sys.exit(-1)
+
+# Add release
+RELEASE = "0.0.0-dev0"
+
+# Check numpy is installed
 try:
+    import numpy as np
+except ImportError:
+    print("MembraneCurvature requires numpy>=1.20.2")
+
+
+if __name__ == "__main__":
+
     with open("README.md", "r") as handle:
         long_description = handle.read()
-except:
-    long_description = "\n".join(short_description[2:])
 
+    # set requirements
+    install_requires = [
+        'numpy>=1.20.2',
+        'mdanalysis>=2.0.0',
```

Comment:
Do these requirements include 2.0.0b, or do you want to exclude 2.0.0b? Look at how version information is resolved by setuptools.

Reply:
It's actually the `2.0.0b0` version. Updated in the last push.